plugin/trino-ldap-group-provider/src/main/java/io/trino/plugin/ldapgroup/LdapGroupProviderConfig.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.ldapgroup;
+
+import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
+import io.airlift.configuration.ConfigSecuritySensitive;
+import jakarta.validation.constraints.NotNull;
+
+public class LdapGroupProviderConfig
+{
+    private String ldapAdminUser;
+    private String ldapAdminPassword;
+    private String ldapUserBaseDN;
+    private String ldapUserSearchFilter = "(uid={0})";
+    private String ldapGroupsNameAttribute = "cn";
+    private boolean ldapUseGroupFilter;
+
+    @NotNull
+    public String getLdapAdminUser()
+    {
+        return ldapAdminUser;
+    }
+
+    @Config("ldap.admin-user")
```

Comment:
How about `ldap.user` - It might not always be an admin user right ? And description could be `User to use when connecting to LDAP server`